src/bundle/Resources/views/locationview.html.twig

```diff
@@ -21,7 +23,7 @@
                     <p class="ez-content-field-name">{{ ez_field_name(content, field.fieldDefIdentifier) }}</p>
                     <div class="ez-content-field-value">
                         {% if ez_is_field_empty(content, field.fieldDefIdentifier) %}
-                            <em>{{"fieldview.field.empty"|trans({}, "fieldview")}}</em>
+                            <em>{{"fieldview.field.empty"|trans({}, "fieldview")|desc('The field is empty')}}</em>
```

Comment:
Shouldn't it be `This field is empty`? That's what seems to be used on the mockups